src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.cs

```diff
@@ -239,51 +237,20 @@ public static int BindExistingObject(object rawObj, int jsId)
                 if (IDFromJSOwnedObject.TryGetValue(o, out result))
                     return result;
 
-                result = NextJSOwnedObjectID++;
+                result = (int)(IntPtr)GCHandle.Alloc(o, GCHandleType.Normal);
```

Comment:
Just noting again that we need the table, gchandle IDs can be reused by the runtime.

Reply:
We know GCHandle could be re-used, we already discussed that. 
But the GCHandle id/index could be only re-used after it was released by us in 'Runtime.ReleaseJSOwnedObjectByHandle()'
Could you please be more specific on how it could go wrong ? Or write unit test for it.